juniorguru_chick/bot.py

```diff
@@ -30,6 +30,14 @@ async def on_message(message: discord.Message) -> None:
         logger.info("Message sent by the bot itself, skipping")
         return
     if message.guild is None:
+        try:
+            response = "Píp píp píp! Jsem jen malé kuřátko, které neumí číst soukromé zprávy a odpovídat na ně.\
```

Comment:
Backslashes are the last resort. This might be better:

```python
long_string = ("this will be"
               " concatenated tog"
               "ether!")
```